Cargo.toml

```diff
@@ -1,10 +1,14 @@
 [package]
 name = "bevy-glsl-to-spirv"
-version = "0.2.0"
+version = "0.2.2"
 authors = ["Pierre Krieger <pierre.krieger1708@gmail.com>", "The vulkano contributors", "Carter Anderson <mcanders1@gmail.com>", "Nicholas Rishel <rishel.nick@gmail.com>"]
 description = "Deprecated. This crate is a temporary measure until native rust shader compilation like https://github.com/gfx-rs/naga lands."
 repository = "https://github.com/cart/glsl-to-spirv"
 license = "MIT/Apache-2.0"
 build = "build/build.rs"
 categories = ["rendering::graphics-api"]
 edition = "2018"
+
+[build-dependencies]
+cmake = "0.1.45"
```

Comment:
This will add these dependencies on all platforms. Can we filter this down to 32bit windows builds?

Reply:
Unfortunately not. Target-dependent build scripts are not supported at the moment - see rust-lang/cargo#4932 and rust-lang/cargo#5892.

Reply:
I've also considered specifying it as a feature, but target-specific features are only enabled on nightly and not yet stabilized - rust-lang/cargo#7914.

Reply:
I just thought of a potential workaround by having subcrates, I'll see if this is a viable approach to remove these dependencies.

Reply:
According to the first issue you mentioned the following **is** supported:

```toml
[target.'cfg(unix)'.build-dependencies.cc]
version = "1.0"
```

Only conditionally using a build script is not currently supported, but we need a build scripts unconditionally anyway.

Reply:
That unfortunately does not solve our issue. `extern crate cmake;` cannot be made conditional upon the target platform for build scripts, only the **host** platform, which means the build dependency effectively must exist unconditionally.